components/Samples/Form.js

```diff
@@ -72,146 +85,173 @@ const SampleForm = () => {
+                touched={touched}
+                errors={errors}
               />
-            </CheckboxGroup>
-          </FormRow>
-          <FormRow>
-            <RadioButtonGroup
-              id="radioGroup"
-              label="One of these please"
-              value={values.radioGroup}
-              error={errors.radioGroup}
-              touched={touched.radioGroup}
-            >
-              <Field
-                component={RadioButtonGroupItem}
-                name="radioGroup"
-                id="radioOption1"
-                label="Choose this option"
+            </FormRow>
+            <FormRow>
+              <FormInput
+                inputType="textarea"
+                fieldName="interests"
+                getFieldProps={getFieldProps}
+                errors={errors}
+                touched={touched}
+                label="Interests"
+                helpText="Maximum of 12 interests.  Type a comma between each interest."
               />
-              <Field
-                component={RadioButtonGroupItem}
-                name="radioGroup"
-                id="radioOption2"
-                label="Or choose this one"
+            </FormRow>
+            <FormRow>
+              <FormInput
+                inputType="markdown"
+                fieldName="bio"
+                getFieldProps={getFieldProps}
+                setFieldTouched={setFieldTouched}
+                setFieldValue={setFieldValue}
+                errors={errors}
+                touched={touched}
+                label="Bio"
               />
-            </RadioButtonGroup>
-          </FormRow>
-          <FormRule />
-          <FormSubmit />
-        </Form>
-      )}
+            </FormRow>
+            <FormRow>
+              <div>
+                <FormInput
+                  fieldName="agreeToCodeOfConduct"
+                  getFieldProps={getFieldProps}
+                  errors={errors}
+                  touched={touched}
+                  label="Agree to <a href=''>Code of Conduct</a>"
+                  inputType="checkbox"
+                />
+              </div>
+              <div>
+                <FormInput
+                  fieldName="agreeToCommitments"
+                  getFieldProps={getFieldProps}
+                  errors={errors}
+                  touched={touched}
+                  label="Agree to commitments to THAT Conference laid out above"
+                  inputType="checkbox"
+                />
+              </div>
+              <div>
+                <FormInput
+                  fieldName="agreeToBeingRecorded"
+                  getFieldProps={getFieldProps}
+                  errors={errors}
+                  touched={touched}
+                  label="Agree to being recorded"
+                  inputType="checkbox"
+                />
+              </div>
+            </FormRow>
+            <FormRow>
+              <CheckboxGroup
+                id="checkboxGroup"
```

Comment:
applies for all in this section but we should have better id values for things like screen readers.

Reply:
although I guess this is a sample file. is this something sticking around or just temp?

Reply:
This was just to get a formik/yup poc out.  It will go away once we have a form merged in to master.